pkg/clusterversion/cockroach_versions.go

```diff
@@ -288,6 +288,16 @@ const (
 	// to be only session based.
 	V24_1_SessionBasedLeasingUpgradeDescriptor
 
+	// V24_1_PebbleFormatSyntheticPrefixSuffix upgrades Pebble's format major version to
+	// FormatSyntheticPrefixSuffix, allowing use of virtual sstables in Pebble.
+	V24_1_PebbleFormatSyntheticPrefixSuffix
+
+	// V23_2_EnablePebbleFormatSyntheticPrefixSuffix enables the Pebble
+	// FormatMajorVersion for synthetic prefix and suffix replacement. Note that
+	// the ratcheting for the format major version in Pebble should have happened
+	// with V24_1_PebbleFormatSyntheticPrefixSuffix above.
+	V24_1_EnablePebbleFormatSyntheticPrefixSuffix
```

Comment:
You can rebase after https://github.com/cockroachdb/cockroach/pull/119491 merges and drop this second version from this commit.

Reply:
while i agree with you that second cluster version is completely useless. A very powerful rep from big storage (@itsbilal ) advised me to add the second setting. We can chat more about this in the meeting today.

Reply:
rebased and removed the second cluster setting.